scanner/src/test/kotlin/scanners/fossid/FossIdTest.kt

```diff
@@ -103,6 +105,18 @@ class FossIdTest : WordSpec({
         mockkStatic("org.ossreviewtoolkit.clients.fossid.ExtensionsKt")
     }
 
+    beforeTest {
+        // Here a static function of the companion object is mocked therefore `mockkobject` needs to be used.
+        // See https://lifesaver.codes/answer/cannot-mockkstatic-for-kotlin-companion-object-static-method-136
+        mockkObject(FossIdRestService)
+
+        every { FossIdRestService.createService(any()) } returns createServiceMock()
+    }
+
+    afterTest {
+        unmockkObject(FossIdRestService)
+    }
+
```

Comment:
Just wondering: How are the new test assets being used, as I don't see any (relevant) changes to the test code?

Reply:
This is because we stopped using `FossIdRestService` and use `FossIdServiceWithVersion` instead.
The latter does a `getFossIdVersion` called that should be mocked with an asset:
https://github.com/bosch-io/oss-review-toolkit/blob/445df9a1354b678a64c5e5956c42efb5ace5bb17/clients/fossid-webapp/src/main/kotlin/FossIdServiceWithVersion.kt#L33

Reply:
Thanks for the explanation!